scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -687,3 +687,53 @@ TDFN-6-1EP_2.5x2.5mm_P0.65mm_EP1.3x2mm:
   suffix: ''
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
+
+TDFN-10-1EP_2x3mm_P0.5mm_EP2.08x0.8mm:
+  device_type: 'TDFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://pdfserv.maximintegrated.com/package_dwgs/21-0429.PDF'
+  # ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x:
+    nominal: 2.0
+    tolerance: 0.1
+  body_size_y:
+    nominal: 3.0
+    tolerance: 0.1
+  overall_height:
+    nominal: 0.75
+    tolerance: 0.05
+
+  lead_width:
+    nominal: 0.25
+    tolerance: 0.05
+  lead_len:
+    nominal: 0.30
+    tolerance: 0.1
+
+  EP_size_x:
+    nominal: 0.9
+    tolerance: 0.1
+  EP_size_y:
+    nominal: 2.0
+    tolerance: 0.1
+
+  EP_num_paste_pads: [2, 2]
```

Comment:
I suggest a single paste pad in x direction.
Currently, there's 90µm clearance between them, probably not doing much good but requiring a more delicate (and expensive) stencil.